content/sensu-go/6.1/plugins/supported-integrations/ansible.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Ansible integration"
+linkTitle: "Ansible"
+description: "Use the Sensu Ansible Handler plugin to integrate Sensu with your existing Ansible workflows. Read about the features of Sensu's Ansible integration and learn how to get the plugin."
+version: "6.1"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.1:
+    parent: supported-integrations
+---
+
+**COMMERCIAL FEATURE**: Access the Sensu Ansible Handler integration in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][6].
+
+The Sensu Ansible Handler plugin is a Sensu [handler][1] that launches Ansible Tower job templates for automated remediation based on Sensu event data.
+
+{{% notice note %}}
+**NOTE**: The Sensu Ansible Handler plugin is an example of Sensu's auto-remediation integrations.
+To find more integrations, search [Bonsai, the Sensu asset index](https://bonsai.sensu.io/).
+{{% /notice %}}
+
+## Features
+
+The Sensu Ansible Handler plugin supports both Ansible Tower and Ansible AWX implementations of the Ansible Tower API, authenticating using Ansible Tower API tokens.  
+
+- Specify Ansible Tower job templates for remediation actions for checks with Sensu annotations.
```

Comment:
@jspaleta is this the same thing as the "optional job template requests" described in the fourth bullet point below?

Reply:
Clarification from Jef:
```
related... the first one is basic functionality...     the second bullet is more expansive...  we could rewrite the first one as...
Specify default Ansible Tower job template for remediation actions for all checks, and then use check annotations to override the default on  a check by check basis if needed.
```